numpy/lib/tests/test_polynomial.py

```diff
@@ -222,6 +222,12 @@ def test_poly_eq(self):
         assert_equal(p == p2, False)
         assert_equal(p != p2, True)
 
+    def test_polydiv(self):
+        b=np.poly1d([2, 6, 6, 1])
+        a=np.poly1d([-1j, (1+2j), -(2+1j), 1])
+        q, r = np.polydiv(b, a)
+        assert_equal(q*a + r, b)
```

Comment:
Can you add an `assert_equal(q.dtype, np.complex128)` and `assert_equal(r.dtype, np.complex128)`?